README.md

```diff
@@ -1,4 +1,7 @@
-## ORCA Static Documentation
+# Overview
+The Operational Recovery Cloud Archive (ORCA) provides a baseline solution for creating, and managing operational backups in the cloud. In addition, best practices and recovery code that manages common baseline recovery scenarios is also maintained. Requirements and use cases for ORCA are derived from the ORCA Working Group.
```

Comment:
will it make sense to also add the link to the orca working group? 
https://wiki.earthdata.nasa.gov/display/CUMULUS/ORCA+Working+Group+Meeting+Notes

Reply:
We could, the down side to this is that is only accessible to people in the ESDIS project as the wiki pages are not public. I think leaving it off in this case might be better since not everyone can access those pages.

Reply:
sounds good